src/promptflow/promptflow/_sdk/entities/_flow/prompty.py

```diff
@@ -0,0 +1,78 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from os import PathLike
+from pathlib import Path
+from typing import Dict, Union
+
+from promptflow._constants import FlowLanguage
+from promptflow._sdk._constants import BASE_PATH_CONTEXT_KEY
+from promptflow.exceptions import ErrorTarget, UserErrorException
+
+from .base import FlowBase
+
+
+class Prompty(FlowBase):
+    def __init__(
+        self,
+        path: Union[str, PathLike],
+        code: Union[str, PathLike],
+        data: dict,
+        **kwargs,
+    ):
+        # prompty file path
+        path = Path(path)
+        # prompty folder path
+        code = Path(code)
+        super().__init__(code=code, path=path, data=data, content_hash=None, **kwargs)
+
+    # region overrides
+
+    @property
+    def language(self) -> str:
```

Comment:
why prompty has language?

Reply:
additional_include and language will be used in batch run and test. The call has been modified and the two properties have been removed